dist/index.html

```diff
@@ -1,9 +1,43 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8">
-    <title>Output Management</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1"><script defer src="runtime.js"></script><script defer src="main.js"></script></head>
-  <body>
-  </body>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LeaderBoard</title>
+<script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+<body>
+    <h1> LeaderBoard</h1>
+
+    <div class="container">
+        <div class="left-contain">
+            <h3> Recent score</h3>
+            <button>Refresh</button>
+            <ul>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, the list items should be added dynamically using javascript. Please populate this part using javascript.